### ISSUE TITLE
Conference alias is always null

### ISSUE THREAD

**Issue by vodemn** [DolbyIO/comms-sdk-flutter#110]
Inviatation does not contain conferenceAlias, so we fetch the conference, but for me the conferenceAlias is always null. What can be the problem?

```dart
@Subscribe(threadMode = ThreadMode.MAIN)
    fun onEvent(event: InvitationReceivedNotificationEvent) {
        val invitation = event.invitation
        val conference = invitation.conferenceId?.let { VoxeetSDK.conference().getConference(it) } ?: return
        result?.success(
            hashMapOf(
                "type" to NotificationEventType.INVITATION_RECEIVED.rawValue,
                "conferenceId" to event.invitation.conferenceId,
                "conferenceAlias" to conference.alias,
                "participant" to event.invitation.inviter.info?.toJson()
            )
        )
    }
    ```

**Comment by graduad** [comment]
Hi @vodemn , thank you for raising this issue. We are looking into it. However the code snipped above looks to be written in the Kotlin language, and since the issue you have raised is against our Flutter SDK repo, I was wondering if you could give us some more context with regard to your setup. Eg. what version of the dolbyio_comms_sdk_flutter are you using? Could you let us know how you have included the afore mentioned flutter plug-in in your app? Thank you very much in advance.

**Comment by vodemn** [comment]
Hello @graduad ! I'm working on my own flutter plugin and I face the issue, that iOS and Web SDKs do have `conferenceAlias` in the invitation, but Android SDK doen't. Fetched conference comes with `conferenceAlias == null`.
I use the following SDKs versions:
- 3.4.0 for Web
- 3.4.0 for iOS
- 3.4.0 for Android

**Comment by FabienLavocat** [comment]
See issue https://github.com/voxeet/voxeet-sdk-android/issues/12

**Comment by graduad** [comment]
Fixed in 3.7.0
